en/docs/reference/connectors/salesforce-connectors/sf-inbound-endpoint-reference-configuration.md

```diff
@@ -7,84 +7,84 @@ The following configurations allow you to configure Salesforce Inbound Endpoint
     <th>Parameter</th>
     <th>Description</th>
     <th>Required</th>
-    <th>Possible Values</th>
+    <th>Example Values</th>
     <th>Default Value</th>
   </tr>
   <tr>
-    <td>sequential</td>
-    <td>Whether the messages should be polled and injected sequentially.</td>
+    <td>interval</td>
+    <td>The polling interval for the Salesforce inbound endpoint.</td>
     <td>Yes</td>
-    <td>true , false</td>
-    <td>TRUE</td>
+    <td>-</td>
+    <td>-</td>
   </tr>
   <tr>
-    <td>replay</td>
-    <td> Enabling this will read the event ID stored in the Registry DB or from the text file stored in the local machine.</td>
+    <td>connection.salesforce.salesforceObject</td>
```

Comment:
```suggestion
    <td><code>connection.salesforce.salesforceObject</code></td>
```

Shall we format the text for the param name, example values, and default value?

Reply:
sure.